pkg/controllers/subnet/subnet_controller.go

```diff
@@ -84,7 +84,7 @@ func (r *SubnetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 				return ResultRequeue, err
 			}
 			if obj.Spec.AccessMode == "" {
-				obj.Spec.AccessMode = v1alpha1.AccessMode(vpcNetworkConfig.DefaultSubnetAccessMode)
+				obj.Spec.AccessMode = v1alpha1.AccessMode(vpcNetworkConfig.DefaultPodSubnetAccessMode)
```

Comment:
Since DefaultPodSubnetAccessMode is only for POD, but subnet is not for POD. 
So, we should not assign vpcNetworkConfig.DefaultPodSubnetAccessMode to subnet AccessMode.
I think we should set obj.Spec.AccessMode as Private.